backend/trojan_dining/tests/test_webscraper.py

```diff
@@ -59,5 +59,4 @@ def test_winter_break(self):
 # ----------------------------------------------
 if __name__ == '__main__':
     unittest.main()
-
-
+    
```

Comment:
Truly the smallest of nits, but lets try to end our files in a newline. It shows the red circle here bc there's a tab or something there. The autoformatter will do this for you if you keep linting enabled